.github/workflows/review.yml

```
@@ -25,87 +25,106 @@ jobs:
         run: |
           mkdir -p tools
           cat > tools/reviewer.js <<'JS'
-// Simple heuristic PR reviewer. Usage inside CI:
-//   git diff --unified=0 BASE HEAD > diff.patch
-//   node tools/reviewer.js --file diff.patch
 import fs from 'fs';
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 28: could not find expected ':'

(syntax-check)